cmd/submit/rbac.go

```diff
@@ -40,7 +40,7 @@ func getRBACCmd(ks meta.IKubescape, submitInfo *v1.Submit) *cobra.Command {
 			k8s := k8sinterface.NewKubernetesApi()
 
 			// get config
-			clusterConfig := getTenantConfig(&submitInfo.Credentials, "", k8s)
+			clusterConfig := getTenantConfig(&submitInfo.Credentials, "", "", k8s)
```

Comment:
I think the custom cluster name should be supported also when running the submit command

Reply:
Hello @dwertent. Can you pls explain how should it work with the submit command ? 
If you are talking about the `rbac` subcommand, then I already see the custom cluster's name on the portal 
(when I run `kubescape submit rbac`)

![0](https://user-images.githubusercontent.com/43821031/188587030-aee3e97a-5920-48ea-bea2-a604dc4d5585.png)

Reply:
You see the custom name since you probably scanned your cluster and add the `--cluster-name` flag